apps/docs/src/app/core/component-docs/tile/examples/action-tile-example.component.html

```diff
@@ -1,6 +1,6 @@
 <fd-tile type="launch" [action]="true">
-    <button fd-button fd-tile-action-close [compact]="true" fdType="transparent" (click)="window.alert('Close clicked')"></button>
-    <button fd-button fd-tile-action-indicator [compact]="true" fdType="transparent" (click)="window.alert('More clicked')"></button>
+    <button fd-button fd-tile-action-close [compact]="true" fdType="transparent" (click)="window.alert('Close clicked')" title="close"></button>
+    <button fd-button fd-tile-action-indicator [compact]="true" fdType="transparent" (click)="window.alert('More clicked')" title="more"></button>
```

Comment:
Aren't you missing aria-label?

Reply:
Aria label are adde when the button Pr is merged(https://github.com/SAP/fundamental-ngx/pull/6385)